configure

```diff
@@ -2429,6 +2447,9 @@ fi
 if test "$valgrind_dev" = "yes"; then
   output_sym "CONFIG_VALGRIND_DEV"
 fi
+if test "$linux_blkzoned_h" = "yes" ; then
+  output_sym "HAVE_LINUX_BLKZONED_H"
```

Comment:
Can you make this CONFIG_LINUX_BLKZONED? I noticed that slipped through in the asprintf/vasprintf patches, and it doesn't match what any of the other configure tests name things.

Reply:
OK, done.